### PR TITLE
Add PdfImages command & tests

### PR DESCRIPTION
**Pull request by ifunk**
Have added passing tests (have tested on Windows but should work cross-platform).

### CODE REVIEW

**Comment by ifunk** [comment]
I have been using PHP-XPDF with the poppler binaries without realising these are two separate projects. Are you explicitly only supporting xpdf?

Perhaps instead I should contribute this to https://github.com/php-poppler/php-poppler

File: tests/XPDF/Tests/PdfImagesTest.php [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+        foreach ($result as $file) {
+            $this->assertEquals(true, file_exists($file));
+            $this->assertEquals(true, preg_match('/\.(ppm)$/', $file));
+        }
+
+        $this->removeTempFiles($result);
+    }
+
+
+//     public function testGetText()
```
Comment (aztech-dev) [inline tests/XPDF/Tests/PdfImagesTest.php]: Please remove the commented code

**Comment by fabre-thibaud** [comment]
Sorry, I messed up. Do you mind fixing the issues I left comments for and re-opening a pull request ? Thanks

**Comment by ifunk** [comment]
Is it an issue that I have changed dependency of xpdf to poppler-utils?

Following semantic versioning would require a major version bump with this change.

**Comment by fabre-thibaud** [comment]
Actually, it would be better to create a build matrix in Travis to run tests with both xpdf and poppler-utils so we can ensure compatibility with both tools. I'm not against adding poppler support, but dropping xpdf is not an option.

**Comment by ifunk** [comment]
That's a good idea. I'm not familiar with how to accomplish this in Travis, and also updating tests so the differences in pdftotext versions don't break your tests. Is this something you can do? Do you need me to do anything else?

**Pull request merged**